spaceship/static/wip-overlay.css

```diff
@@ -0,0 +1,78 @@
+#overlay-background {
+  background-color: rgb(0, 0, 0);
+  display: none;
+  left: 0px;
+  top: 0px;
+  height: 100%;
+  margin: 0px;
+  opacity: 0.75;
+  position: fixed;
+  width: 100%;
+  z-index: 999998;
+}
+.overlay-container {
+    box-sizing: border-box;
+    display: none;
+    margin: 0px auto;
+    padding: 15px 15px 15px 15px;
+    max-width: calc(100% - 30px);
+    position: fixed;
+    opacity: 1;
+    z-index: 999999;
+    left: 0px;
+    right: 0px;
+    width: 678px;
+    height: calc(100% - 20px);
+    top: 50%;
+    transform: translateY(-50%);
+    max-height: 510px;
+    background: rgba(255,255,255, 0.95);
+    h1,h2,h3,p,li,a {
+      font-family: "Circular Std", "Lato", sans-serif;
+      -webkit-font-smoothing: antialiased;
+      -moz-osx-font-smoothing: grayscale;
+    }
+    @media screen and (max-width: $screen-xs-min) {
```

Comment:
Alas vanilla css doesn't support named constants like this (this looks like it may be another sass thing)? media queries would also have to go at the top level, and then you can specify a selector inside them.

Reply:
Very likely the file I was looking at for inspiration must have been for a CSS preprocesser :/